app/DoctrineMigrations/Version139.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version139 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE TABLE report_submission (id SERIAL NOT NULL, report_id INT DEFAULT NULL, archived_by INT DEFAULT NULL, created_by INT DEFAULT NULL, archived BOOLEAN DEFAULT \'false\' NOT NULL, created_on TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id))');
```

Comment:
Do we really need archived BOOLEAN DEFAULT \'false\'  ? The archived by will indicate this, if set then archived. If not set then not archived?